docs/sources/release-notes/v1-10.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Version 1.10 release notes
+menuTitle: V1.10
+description: Release notes for Grafana Pyroscope 1.10
+weight: 400
+---
+
+# Version 1.10 release notes
+
+The Pyroscope team is excited to present Grafana Pyroscope 1.10.
+
+This version of Pyroscope adds experimental support for OpenTelemetry profiles ([#2177](https://github.com/grafana/pyroscope/pull/2177)).
+The OTLP protocol is not stable. For more information about the protocol, refer to [OpenTelemetry announces support for profiling](https://opentelemetry.io/blog/2024/profiling/).
+
+In addition, this release improves stability, performance, and documentation.
+
+Notable changes are listed below. For more details, check out the [1.10.0 changelog](https://github.com/grafana/pyroscope/compare/v1.9.0...v1.10.0).
+
+## Improvements and updates
+
+* Use `profilecli admin blocks query` to execute queries directly to single or multiple blocks hosted in your local host or remote bucket. ([#3618](https://github.com/grafana/pyroscope/pull/3618), [#3625](https://github.com/grafana/pyroscope/pull/3625), [#3610](https://github.com/grafana/pyroscope/pull/3610))
```

Comment:
Do we have documentation for this command? There is good documentation in the PRs for #3618 and #3610. However, with the refactor from query-blocks into the admin blocks query, that negates some of the examples from the PR descriptions. I had added the content from 3618 and 3610 to the profilecli doc and then removed it after I saw the refactor. 

@alsoba13   any suggestions?